src/tests/ftest/util/cmocka_utils.py

```diff
@@ -28,7 +29,7 @@ def get_cmocka_command(command, parameters=None):
     """
     keywords = ["Process received signal", "stack smashing detected", "End of error message",
                 "buffer overflow detected"]
-    path, executable = os.path.split(command)
+    path, executable = os.path.split(command.strip())
```

Comment:
This is still not working:
```
2025-02-28 23:57:51,579 run_utils        L0471 DEBUG| Running on wolf-309 with a 60 second timeout: /usr/bin/pgrep --list-full --full -x '/usr/bin/daos_test -n /var/tmp/daos_testing/configs/daos_control.yml -m '
2025-02-28 23:57:51,793 run_utils        L0333 DEBUG|   wolf-309 (rc=1): <no output>
```
from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15993/2/artifact/Functional%20Hardware%20Medium%20Verbs%20Provider/daos_test/suite.py/job.log

Reply:
This is due to the extra space being introduced by the `parameters` argument.  See updated changes.